### PR TITLE
DOCS-2045: Address trigger cloud sync feedback

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/data/trigger-sync.md [hunk 1]
```diff
@@ -1,31 +1,43 @@
...
 type: "docs"
 tags: ["data management", "cloud", "sync"]
 # SME: Alexa Greenberg
 ---
 
-If you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
+You can use a {{< glossary_tooltip term_id="module" text="module" >}} to sync data only when a certain logic condition is met, instead of at a regular time interval.
+For example, if you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
+Or, you may want to trigger sync only when your machine detects an object of a certain color.
+The code for both of these examples is provided by the [trigger-sync-examples module](https://github.com/viam-labs/trigger-sync-examples-v2), and you can [create your own module](/registry/create/) if you want to use different logic.
```
Comment (npentrel) [inline docs/data/trigger-sync.md:14]: Both of which examples? reference unclear.

**Review by npentrel — COMMENTED** [review]
One comment, otherwise good to go!

File: docs/data/trigger-sync.md (hunk 1 above)
Comment (JessamyT) [reply docs/data/trigger-sync.md:14]: Does this work?
```suggestion
The code for the timeframe and color detection trigger examples is provided by the [trigger-sync-examples module](https://github.com/viam-labs/trigger-sync-examples-v2), and you can [create your own module](/registry/create/) if you want to use different logic.
```

Comment (npentrel) [reply docs/data/trigger-sync.md:14]: I am only just realizing that this maps to the first two sentences here which are about time frame and color detection example. I was somehow seeing that as disconnected... The original copy is fine actually - sorry!

**Review by npentrel — APPROVED** [review]

File: docs/data/trigger-sync.md (hunk 1 above)
Comment (JessamyT) [reply docs/data/trigger-sync.md:14]: Okay great :)

**Pull request merged**